src/endpoints/getPlayerStats.ts

```diff
@@ -0,0 +1,59 @@
+import FullPlayerStats from '../models/FullPlayerStats'
+import Team from '../models/Team'
+import HLTVConfig from '../models/HLTVConfig'
+import { fetchPage } from '../utils/mappers'
+
+const getPlayerStats = (config: HLTVConfig) => async ({ id , startDate, endDate }: { id: number, startDate: string, endDate: string }): Promise<FullPlayerStats> => {
+    
+
+    var options = '';
+    if (startDate != null && endDate != null) {
+        options = '?startDate='+startDate+'&endDate='+endDate;
+    }
+    const $ = await fetchPage(`${config.hltvUrl}/stats/players/${id}/-${options}`, config.loadPage)
+
+    const name = $('.statsPlayerName').text() || undefined
+    const ign = $('.context-item-name').text();
+
+    const image = $('.context-item-image').attr('src');
+
+    const getInfo = function(i) { return $($('.divided-row').find('.large-strong').get(i)); };
```

Comment:
Prefer arrow functions instead of function expressions, e.g. `const getInfo = (i) => ...`